easybuild/framework/easyconfig/types.py

```diff
@@ -85,9 +99,10 @@ def is_value_of_type(value, typ_spec):
                 else:
                     raise EasyBuildError("Unknown type requirement specified: %s", er_key)
```

Comment:
please add an extra log message that mentions whether the overal type check passed, like there was in `check_type_of_param_value`